cost/google/schedule_instance/README.md

```diff
@@ -0,0 +1,78 @@
+# Google Schedule Instance
+
+## What it does
+
+This Policy Template allows you to schedule start and stop times for your instance, along with the option to terminate instance, update and delete schedule.         
```

Comment:
I think this area should include the cloud the policy supports.  I know the Heading includes it, but this should include Google also.  Same for the short_description on the policy_template